### DISCUSSION TITLE
How to customize Rich Edior Size

### DISCUSSION THREAD

**Discussion by Aung-Zaw-Phyo** [filamentphp/filament#13352] [Help]
### Package

Panel builder

### Package Version

3

### How can we help you?

I want to resize smaller one

**Comment by xalabama** [comment]
You can try to use the method 'extraInputAttributes'. This allows you to attach classes or other attributes to the trix editor tag.

```php
RichEditor::make('editor')
    ->extraInputAttributes(['class' => 'your-class'])
```

**Reply by Aung-Zaw-Phyo** [reply]
Thanks you
